src/client/app/effects/github.ts

```diff
@@ -9,9 +9,10 @@ import { Effect, Actions } from '@ngrx/effects';
 import { Http, ResponseContentType } from '@angular/http';
 import * as clipboard from 'clipboard';
 import { UIEffects } from './ui';
-import { find } from 'lodash';
+import { find, isNil } from 'lodash';
```

Comment:
I personally like the isNil simply for its explicitness (I always worry about seeing a double-equals, and feeling like maybe it will trigger for other falsy things, not just null and undefined)